wpa_supplicant/wpa_cli_zephyr.c

```diff
@@ -83,7 +83,7 @@ static int _wpa_ctrl_command(struct wpa_ctrl *ctrl, const char *cmd, int print,
 	if (print) {
 		buf[len] = '\0';
 		if (buf[0] != '\0')
-			wpa_printf(MSG_DEBUG, "%s", buf);
+			wpa_printf(MSG_INFO, "%s", buf);
```

Comment:
This would mean more noise for other commands, instead of print can't we prepare a structure with response and send it back? See `z_wpa_ctrl_status` as an example.

Reply:
I don't think response info is noise.
I think command with interactive means they need to get response shown.
DEBUG level means this log is for debug, but I think this log is for user info.

Reply:
In the upstream, wpa_cli is an interactive application that uses control interface, but in Zephyr downstream the control interface is not interactive, so, all responses are parsed and returned to the interested caller via structures programatically.

Reply:
Changed the commit message to try to solve the misleading here.

Reply:
Thanks for clarifying.